pkg/utils/utils.go

```diff
@@ -92,6 +92,17 @@ func GetPfName(pciAddr string) (string, error) {
 	}
 
 	path := filepath.Join(sysBusPci, pciAddr, "physfn", "net")
+	vertIOpath := filepath.Join(sysBusPci, pciAddr, "physfn")
+	dir, err := os.ReadDir(vertIOpath)
+	if err != nil {
+		return "", nil
```

Comment:
Maybe we want to return the error
```suggestion
		return "", fmt.Errorf("can't open path %s: %w", virtIOpath, err)
```